java/src/jmri/util/swing/JmriMouseListener.java

```diff
@@ -0,0 +1,78 @@
+package jmri.util.swing;
+
+import java.awt.event.MouseEvent;
+
+/**
+ * Replacement for MouseListener.
+ * This class is used to replace MouseEvent with JmriMouseEvent.
```

Comment:
Minor thing, no need to change, but making the reference to JmriMouseEvent as an `@link` helps the Javadoc a bit.

Reply:
Thanks. Done.
I hope I got it right.